problems/objects.js

```diff
@@ -187,16 +274,48 @@ const countOccurance = () => {};
  * @returns {Object} Counts all characters except spaces  {a: 2, g: 1, o: 2, d:1, s: 1, n:a, k:1, e: 1}
  */
 
-const countOccuranceNoSpaces = () => {};
+const countOccuranceNoSpaces = (str) => {
+  let obj = {}
+  let arr = str.split(" ")
+  let newStr = arr.join("")
+  for (let i = 0; i < newStr.length; i++) {
+    const element = newStr[i].toLowerCase();
+    if (obj[element]) {
+      obj[element]++
+    } else {
+      obj[element] = 1
+    }
+  }
+  return obj
+};
 
 /**
  * Takes in an array and returns the most common element.
  * @param {Array} array - array of elements
  * @returns {(number|string)} Most common element
  */
 
-const mostCommonElement = () => {};
+const mostCommonElement = (array) => {
+  let obj = {}
+  let mostCommon = 0
+  let nameToReturn = " "
+  for (const i of array) {
```

Comment:
It's not standard to use the variable 'i' in a 'for of' loop because the variable here is not the index of the array, this variable is the element in the array. It's more common to use 'el' or 'element'.